client/web/src/components/ActivationDropdown/ActivationDropdown.story.tsx

```diff
@@ -88,20 +102,35 @@ _04Completed.parameters = {
     },
 }
 
-export const _14Completed: Story = () => (
+export const _14Completed: Story = args => (
     <ActivationDropdown
         {...commonProps}
         activation={{
             ...baseActivation(),
             completed: {
-                ConnectedCodeHost: boolean('ConnectedCodeHost', true),
-                DidSearch: boolean('DidSearch', false),
-                FoundReferences: boolean('FoundReferences', false),
-                EnabledRepository: boolean('EnabledRepository', false),
+                ...args,
             },
         }}
     />
 )
+_14Completed.argTypes = {
+    ConnectedCodeHost: {
+        control: { type: 'boolean' },
+        defaultValue: true,
+    },
+    DidSearch: {
+        control: { type: 'boolean' },
+        defaultValue: false,
+    },
+    FoundReferences: {
+        control: { type: 'boolean' },
+        defaultValue: false,
+    },
+    EnabledRepository: {
```

Comment:
```suggestion
    EnabledSharing: {
```

This control was also affected by the previous comment. 🙂